### PR TITLE
improve candle time uniformization and uniformize klines time

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/producers/kline_updater.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ def __create_time_frame_kline_task(self, time_frame):
                                                                                             limit=self.KLINE_LIMIT)
                     try:
                         candle = candle[0]
+                        self.channel.exchange_manager.uniformize_candles_if_necessary(candle)
```
Comment (Herklos) [inline octobot_trading/producers/kline_updater.py]: :+1:

File: octobot_trading/producers/ohlcv_updater.py [hunk 2]
```diff
@@ -68,6 +68,7 @@ def _create_pair_candle_task(self, pair):
         # fetch history
         candles: list = await self.channel.exchange_manager.exchange \
             .get_symbol_prices(pair, time_frame, limit=self.OHLCV_OLD_LIMIT)
+        self.channel.exchange_manager.uniformize_candles_if_necessary(candles)
```
Comment (Herklos) [inline octobot_trading/producers/ohlcv_updater.py]: :+1:

File: octobot_trading/producers/ohlcv_updater.py [hunk 3]
```diff
@@ -84,7 +85,7 @@ def _create_pair_candle_task(self, pair):
 
                 if candles:
                     last_candle: list = candles[-1]
-                    self.channel.exchange_manager.uniformize_candles_if_necessary(last_candle)
+                    self.channel.exchange_manager.uniformize_candles_if_necessary(candles)
```
Comment (Herklos) [inline octobot_trading/producers/ohlcv_updater.py]: Is `candles[-1]` modified by `uniformize_candles_if_necessary` correctly update `last_candle` value ?

Comment (GuillaumeDSM) [reply octobot_trading/producers/ohlcv_updater.py]: yes

Comment (Herklos) [reply octobot_trading/producers/ohlcv_updater.py]: OK nice

**Pull request merged**